fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/PushDownPredicateScanRule.java

```diff
@@ -47,8 +48,9 @@ public PushDownPredicateScanRule(OperatorType type) {
 
         ScalarOperator predicates = Utils.compoundAnd(lfo.getPredicate(), logicalScanOperator.getPredicate());
         ScalarRangePredicateExtractor rangeExtractor = new ScalarRangePredicateExtractor();
-        predicates = rangeExtractor.rewriteOnlyColumn(Utils.compoundAnd(Utils.extractConjuncts(predicates)
-                .stream().map(rangeExtractor::rewriteOnlyColumn).collect(Collectors.toList())));
+        predicates = rangeExtractor.rewriteAll(Utils.compoundAnd(Utils.extractConjuncts(predicates)
+                .stream().map(rangeExtractor::rewriteAll).collect(Collectors.toList())));
```

Comment:
There should use `rewriteOnlyColumn` in there, not `rewriteAll`, we can't promise rewrite function will improve performance now